frontend/src/hooks/useSubmitHandler.ts

```diff
@@ -20,7 +20,8 @@ export const useSubmitHandler = () => {
         setSubmitting(false);
       }
 
-      if (!result?.status.isInBlock) return;
```

Comment:
@gaboesquivel we shouldn't comment out broken functionality. 

@DoubleOTheven please look into why `ContractSubmittableResult` seems to incorrectly extend `SubmittableResult` in useInk.

[Here](https://github.com/polkadot-js/api/blob/c028c6958211832bae5e5dda1868466a48195415/packages/api/src/submittable/Result.ts#L37) is the expected shape.

If you undo the comments in this file, you will get build errors:

```
error TS2339: Property 'events' does not exist on type 'ContractSubmittableResult'.
error TS2339: Property 'dispatchError' does not exist on type 'ContractSubmittableResult'.
```

Reply:
good catch, I didn't realized that was missing